libs/template/presets/storefront/experience/experience-data.ts

```diff
@@ -3,6 +3,7 @@ import { provideExperienceData } from '@spryker-oryx/experience';
 import { FooterTemplate } from './footer';
 import { HeaderTemplate } from './header';
 import {
+  accountPage,
```

Comment:
Can you please create `StaticExperienceFeatureLatest` and `StaticExperienceFeature1_1`?
...and use them in `storefrontFeaturesLatest` and `storefrontFeatures1_1`.

Reply:
Will be resolved when the [other PR](https://github.com/spryker/oryx/pull/697) gets merged.

Reply:
It looks like this is still not fixed.

Reply:
Now using the `featureVersion` flag.